apps/api/common/models/tp-job-listing.js

```diff
@@ -8,10 +8,16 @@ const app = require('../../server/server')
 module.exports = function (TpJobListing) {
   TpJobListing.observe('before save', function updateTimestamp(ctx, next) {
     const currentDate = new Date()
+    const expiryDate = new Date()
+    // MVP expiry date defaults to 30 days in the future
+    expiryDate.setDate(expiryDate.getDate() + 30)
```

Comment:
Suggest to remove, in favour of making the field in the front-end required and pre-filled with a default date.